opentelekomcloud/services/swr/resource_opentelekomcloud_swr_domain_v2.go

```diff
@@ -114,15 +112,20 @@ func resourceSwrDomainRead(_ context.Context, d *schema.ResourceData, meta inter
 		return fmterr.Errorf(ClientError, err)
 	}
 
-	domain, err := domains.Get(client, organization(d), repository(d), d.Id()).Extract()
+	opts := domains.GetOpts{
+		Namespace:    d.Get("organization").(string),
+		Repository:   d.Get("repository").(string),
+		AccessDomain: d.Get("access_domain").(string),
```

Comment:
`AccessDomain` wasn't present in previous version, did something change?

Reply:
it passed directly to function now it in opts